src/docdb/getCosmosClient.ts

```diff
@@ -4,8 +4,14 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { CosmosClient } from "@azure/cosmos";
+import * as https from "https";
+import * as vscode from 'vscode';
 import { appendExtensionUserAgent } from "vscode-azureextensionui";
+import { ext } from "../extensionVariables";
+
+export function getCosmosClient(endpoint: string, key: string, isEmulator: boolean | undefined): CosmosClient {
+
+    const vscodeStrictSSL: boolean | undefined = vscode.workspace.getConfiguration().get<boolean>(ext.settingsKeys.vsCode.proxyStrictSSL);
+    return new CosmosClient({ endpoint, key, userAgentSuffix: appendExtensionUserAgent(), agent: new https.Agent({ rejectUnauthorized: !isEmulator || vscodeStrictSSL }) });
```

Comment:
Before, this was `!isEmulator && vscodeStrictSSL`, and I think it should stay that way. It's reasonably safe to ignore the `vscodeStrictSSL` setting (which will almost always be true) in the emulator case because it should only be traffic from their local machine.

Reply:
Yes, that's true. It make sense to have `!isEmulator && vscodeStrictSSL`.